[BUG] Refresh section about refining Timeline results (#2204)
elastic/security-docs: docs/events/timeline-ui-overview.asciidoc

[[refine-timeline-results]]
== View and refine Timeline results

> Go to the right of the date and time picker, then click *Data view*. To hide raw events and display <<det-engine-terminology, detection alerts>> only, select *Show only detection alerts*. This setting is not selected by default. Keep it de-selected to display both raw events and detection alerts in Timeline.

Reviewer: - The previous version had a nice summary of what exactly this setting does, so it might help to lead with something similar. Otherwise this just starts with instructions that the user might not know why they should follow.
- No need to explain that the user has to move their mouse to the right of the time picker; just say the drop-down is to the right.
- It's a little confusing to first tell the user "select this checkbox" and then in the same paragraph say "keep this deselected." We can tell them what the checkbox does when they select it, and it's pretty clear what happens when it's not selected.
```suggestion
You can select whether Timeline displays <<det-engine-terminology, detection alerts>> and other raw events, or just alerts. By default, Timeline displays both raw events and alerts. To hide raw events and display alerts only, click *Data view* to the right of the date and time picker, then select *Show only detection alerts*.
```

Author: @joepeeples all good points and I can get behind these changes. I figured the section title provided enough lead-in but I can see how setting the stage a bit more (like you did) creates a friendlier tone. I also like the way you ordered the information in your revision. It makes sense to start off with a general statement and then described the default state/setting and how to change it.
